ocfweb/docs/docs/staff/scripts/sorry.md

```diff
@@ -0,0 +1,28 @@
+[[!meta title="sorry: ban/disable an OCF account"]]
+
+## Introduction
+
+`sorry` is the command used to ban or disable OCF accounts. Accounts are banned 
```

Comment:
Banning is different from sorrying. As you'll remember from last semester, we now have have [an entire policy that describes banning](https://www.ocf.berkeley.edu/docs/membership/banning/). We generally never ban a user, so I would not mention it anywhere on this page.